src/render/gpu/SDL_shaders_gpu.c

```diff
@@ -248,7 +225,6 @@ SDL_GPUShader *GPU_GetFragmentShader(GPU_Shaders *shaders, GPU_FragmentShaderID
 void GPU_FillSupportedShaderFormats(SDL_PropertiesID props)
 {
     SDL_SetBooleanProperty(props, SDL_PROP_GPU_DEVICE_CREATE_SHADERS_SPIRV_BOOLEAN, HAVE_SPIRV_SHADERS);
-    SDL_SetBooleanProperty(props, SDL_PROP_GPU_DEVICE_CREATE_SHADERS_DXBC_BOOLEAN, HAVE_DXBC50_SHADERS);
```

Comment:
Do we need HAVE_DXBC51_SHADERS, set for D3D12?

Reply:
It's an either-or thing. It might be reasonable to use DXBC for the render backend instead of DXIL but it's not an urgent change.